spec/scenic/adapters/postgres/refresh_dependencies_spec.rb

```diff
@@ -3,52 +3,76 @@
 module Scenic
   module Adapters
     describe Postgres::RefreshDependencies, :db do
-      it "refreshes dependecies in the correct order" do
-        adapter = Postgres.new
+      let(:adapter) { Postgres.new }
```

Comment:
I usually don't like to use `let` and `before`, but I took a shot at rewriting our way out of them and I hated the results more. Seems like a good `let` and `before`.